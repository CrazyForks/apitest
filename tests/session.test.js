const { spwanTest } = require("./utils");

describe("session", () => {
  test("session", async () => {
    const { stdout, code } = await spwanTest("session", ["--reset"], {
      "FOO": "bar",
    });
    expect(code).toEqual(0);
    expect(stdout).toMatchSnapshot();
  });
});
