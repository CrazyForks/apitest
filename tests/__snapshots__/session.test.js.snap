// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`session session 1`] = `
"module mod1
  unit test1 ✔
  group group1
    unit test1 ✔
module mod2
  unit test1 ✔
  group group1
    unit test1 ✔
    unit test2 ✔
    access variables ✔
  group mod2
    no override parent ✔
  group env1
    env variables ✔
"
`;
